elucidator/src/designation.rs

```diff
@@ -378,11 +378,15 @@ impl DesignationSpecification {
         Ok(map)
     }
 
-    pub fn to_string(&self) -> String {
-        self.members.iter()
+}
+
+impl std::fmt::Display for DesignationSpecification {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let m =self.members.iter()
             .map(MemberSpecification::to_string)
             .collect::<Vec<String>>()
-            .join(", ")
+            .join(", ");
```

Comment:
I'm pretty sure you want your map to use the newly implemented `Display` for `MemberSpecification` rather than `::to_string`

Reply:
Never mind, I didn't know that `Display` automatically implements `ToString`: https://doc.rust-lang.org/std/string/trait.ToString.html